.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+repos:
```

Comment:
can you also add https://github.com/pre-commit/mirrors-mypy ?

Reply:
Added in https://github.com/keisheiled/flake8-implicit-str-concat/pull/16/commits/02c7c9b67b89a3a2eb094fc0610bf152d4a0d24b.

* https://github.com/hugovk/flake8-implicit-str-concat/runs/326393687